jetstream/cli.py

```diff
@@ -143,7 +155,14 @@ def execute(
                         spec.merge(external_spec)
 
                 config = spec.resolve(experiment)
-                self.analysis_class(self.project_id, self.dataset_id, config).run(date)
+                analysis = self.analysis_class(self.project_id, self.dataset_id, config)
+
+                max_run_date = max(worklist_group, key=lambda x: x[1])[1]
+
+                analysis.ensure_enrollments(experiment, max_run_date, self.recreate_enrollments)
+
+                for item in worklist_group:
+                    analysis.run(item[1])
```

Comment:
This part is executed when running jetstream locally without Argo (we still need to ensure that we have enrollments tables in this case) and also every pod started by Argo is using `SerialExecutorStrategy`. In these cases each pod will check if the enrollments tables are available, which they should be since there is the workflow steps for ensuring enrollments are executed before the analysis steps.